templates/archives/stories/geo_timeline.jinja2

```diff
@@ -0,0 +1,101 @@
+        Documentation, proposing remote data access to his and Henri La Fontaine's Mundaneum though telephones and televisions globally. The Mundaneum was a global library museum housed in Cinquantenaire Park, which was shut down in 1934 and then removed by the Germans during their invasion of Belgium in 1940.</p>
+
+    <p></p>
+    <div class="text-center">
+<h1>1937</h1>
+    </div>
+    <div class="text-center">
+<figure class="figure">
+    <img src="{{ static('img/stories/computational_history_1937.jpg') }}"
+         class="class="figure-img img-fluid rounded"
+         alt="Map of United States" />
+    <figcaption class="figure-caption">United States 1937</figcaption>
+</figure>
+    </div>
+    <p>Red dot- New York City, United States, 1937- George Stibitz proved that boolean logic could
+        be used in computers through his Model "K" Adder circuit, which utilized electromechanical relays.</p>
+
+    <p></p>
+    <div class="text-center">
+<h1>1938</h1>
+    </div>
+    <div class="text-center">
+<figure class="figure">
+    <img src="{{ static('img/stories/computational_history_1938.jpg') }}"
+         class="class="figure-img img-fluid rounded"
+         alt="Map of Germany" />
+    <figcaption class="figure-caption">Germany 1938</figcaption>
+</figure>
+    </div>
+    <p>Blue dot- Berlin, Germany, 1938- Konrad Zuse finishes the Z1 computer, the first computer in
+        the world. The Z1 operated on Boolean logic. This prototype, along with Z2 and Z3, were destroyed during WWII.</p>
+<p>Red dot- Mansfield, United States, 1938- Elektro, a smoking, walking, and talking robot, is
+    finished in Ohio by Westinghouse. Elektro was voice controlled, although the voice commands were understood by their rhythm- the robot did not understand their meaning. Elektro was demonstrated in the 1939 World's Fair in New York with great success.</p>
+
+    <p></p>
+    <div class="text-center">
+<h1>1939</h1>
+    </div>
+    <div class="text-center">
+<figure class="figure">
+    <img src="{{ static('img/stories/computational_history_1939.jpg') }}"
+         class="class="figure-img img-fluid rounded"
+         alt="Map of Germany and the United States" />
+    <figcaption class="figure-caption">Germany and the United States 1939</figcaption>
+</figure>
+    </div>
+    <p>Blue dot- Berlin, Germany, 1939- Konrad Zuse develops the Z2. This computer was relay
+        based as an improvement on the Z1, which did not work very well. It was almost done in 1939, but it was delayed to 1940 due to Zuse being required to join the army.</p>
+    <p>Red dot- New York City, United States, 1939- George Stibitz produces the Model I Complex
+        Calculator, the first electrical digital computer. It built on the boolean logic circuits Stibitz created. This computer utilized telephone sitches and electromechanical relays.</p>
+
+    <p></p>
+    <div class="text-center">
+<h1>Sources</h1>
```

Comment:
In the sources I would add which ones are CC, you can just say CC BY-SA 3.0 in parentheses next to the relevant sources with a link to the URL you shared with me (https://creativecommons.org/licenses/by-sa/3.0/)

Reply:
The only source that is CC is the source of the data that is cited. However, I'm unsure how to share the data. For the other sources, does this mean I can just leave them as links?